src/Symfony/Component/Serializer/SerializerAwareInterface.php

```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Component\Serializer;
 
 /**
- * Defines the interface of encoders.
+ * Class accepting a serializer.
```

Comment:
I would remove the comment instead.

Reply:
@fabpot would you remove the comment in NormalizerAwareInterface and DenormalizerAwareInterface aswell then ?

Reply:
If they don't provide significant value, yes.

Reply:
Alright, expanded the removal to few other files